../../../pyodbc.pyi,sha256=cLuLanSS0Vry9aUUvn-i1lpDxHiDcMQwQFypGWqD__E,13127
pyodbc-4.0.34.dist-info/INSTALLER,sha256=zuuue4knoyJ-UwPPXg8fezS7VCrXJQrAP7zeNuwvFQg,4
pyodbc-4.0.34.dist-info/LICENSE.txt,sha256=mFnIV3DJG3hmsPdpyTjjbWTQJgFBtSm5R7Tq2WiD6rQ,868
pyodbc-4.0.34.dist-info/METADATA,sha256=9OvmhDABSFHthoXvVk6wdB2IT48YRKR76m_UntEfUBE,2719
pyodbc-4.0.34.dist-info/RECORD,,
pyodbc-4.0.34.dist-info/REQUESTED,sha256=47DEQpj8HBSa-_TImW-5JCeuQeRkm5NMpJWZG3hSuFU,0
pyodbc-4.0.34.dist-info/WHEEL,sha256=pfjXB0CCNW4PSSqQc2t4Up6p3o0jxBnHy_2o38FQkyE,109
pyodbc-4.0.34.dist-info/direct_url.json,sha256=0AFuhbarjIVOcf0nBBpILnkACA6E4uDdmYHWXNMIXdc,145
pyodbc-4.0.34.dist-info/top_level.txt,sha256=30jvYcCul6_wA26OT5c068GAHzqQLpd9WfQ0mSSsDTU,7
pyodbc.cpython-39-darwin.so,sha256=gUVfwXqvsB9donsAmopqv3Q_KFovaVSPoefJks3R2sk,150156
